src/content/configuration/index.md

```diff
@@ -23,25 +24,11 @@ All the available configuration options are specified below.
 
 T> New to webpack? Check out our guide to some of webpack's [core concepts](/concepts) to get started!
 
-
-## Use different config file
-
-If for some reason you want to use different config file depending on certain situations you can change this via command line by using the `--config` flag.
-
-**package.json**
-
-```json
-"scripts": {
-  "build": "webpack --config prod.config.js"
-}
-```
-
-
 ## Options
 
 Click on the name of each option in the configuration code below to jump to the detailed documentation. Also note that the items with arrows can be expanded to show more examples and, in some cases, more advanced configuration.
 
-W> Notice that throughout the configuration we use Node's built-in [path module](https://nodejs.org/api/path.html) and prefix it with the [__dirname](https://nodejs.org/docs/latest/api/globals.html#globals_dirname) global. This prevents file path issues between operating systems and allows relative paths to work as expected. See [this section](https://nodejs.org/api/path.html#path_windows_vs_posix) for more info on POSIX vs. Windows paths.
+W> Notice that throughout the configuration we use Node's built-in [path module](https://nodejs.org/api/path.html) and prefix it with the [\_\_dirname](https://nodejs.org/docs/latest/api/globals.html#globals_dirname) global. This prevents file path issues between operating systems and allows relative paths to work as expected. See [this section](https://nodejs.org/api/path.html#path_windows_vs_posix) for more info on POSIX vs. Windows paths.
```

Comment:
why do we need that change ? `[__dirname] => [\_\_dirname]`

Reply:
I guess prettier changed it accidentally, will revert later today

Reply:
ok, i think its good to go, fernando doesnt have time to review this so ping me